src/components/Chatwoot.tsx

```diff
@@ -36,8 +39,16 @@ export const ChatwootButton: React.FC = () => {
   }, [])
 
   return chatWootEnabled ? (
-    <Button position='absolute' right='1rem' bottom='1rem' onClick={handleChatWoot}>
-      Get support
-    </Button>
+    <IconButton
+      aria-label='Get support'
+      icon={faMessage}
+      width='48px'
+      height='48px'
+      borderRadius='full'
+      boxShadow='lg'
+      onClick={handleChatWoot}
+      colorScheme='blue'
```

Comment:
I wouldn't use the blue colorscheme, maybe the gray one with like the `outline` variant with a background color to avoid making it transparent

Maybe something like: (didn't try it thought)
```suggestion
      colorScheme='gray'
      variant='outline'
      bg='whiteAlpha.50'
```

Reply:
Here's the goer! 
<img width="136" alt="image" src="https://github.com/user-attachments/assets/416c4074-ce91-41ae-9b91-7e81917eeef2" />